docs/api/cb_util.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`certbot.cb_util`
```

Comment:
This file should be deleted as we didn't rename `le_util` to `cb_util` to prevent breakage.